AIMSICD/src/main/res/values-es/translatable_strings.xml

```diff
@@ -236,11 +236,11 @@
     <string name="describe_the_problem_you_had">¡ ATENCIÓN! Por favor describa el problema que tiene antes de enviarnos estos registros. NO HAY AYUDA SIN DESCRIPCIONES!\n</string>
     <string name="status">Estado:</string>
     <string name="idle">PARADO</string>
-    <string name="normal">NORMAL</string>
+    <string name="ok">OK</string>
```

Comment:
That is exactly what i meant.
I'll do that for you if you aren't sure how to do it